frame/society/src/migrations.rs

```diff
@@ -181,7 +198,7 @@ pub(crate) mod old {
 }
 
 pub fn can_migrate<T: Config<I>, I: Instance + 'static>() -> bool {
-	old::Members::<T, I>::exists()
+	Members::<T, I>::iter().next().is_none()
```

Comment:
Rococo `old::Members` is empty, making `can_migrate` return a false negative.

Reply:
Now you only return true if the list is empty? I'm confused.

Reply:
`Members` are the *new* members. It returns true if new members are empty, signifying the old members have not yet been migrated.